src/xbps/packages/management.md

```diff
@@ -13,6 +13,13 @@ Most general package management is done with the following commands:
    configuration steps for installed packages, and can be used to reconfigure
    certain packages after changes in their configuration files. The latter
    usually requires the `--force` flag.
+- [xbps-alternatives(1)](https://man.voidlinux.org/xbps-alternatives.1) lists or
+   sets the alternatives provided by installed packages. Alternatives is a
+   system which allows the user to have several packages providing the same
+   functionality, such as a given program, and manage which one of those
+   packages will provide said functionality.
```

Comment:
How about:

Alternatives is a system which allows multiple packages to provide common functionality through otherwise conflicting files, by creating symlinks from the common paths to package-specific versions that are selected by the user.

Reply:
Great!